.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug Report
+description: Report a bug with PDF.js
+title: "[Bug]: "
+body:
+  - type: textarea
+    attributes:
+      label: Attach (recommended) or Link to PDF file
+    validations:
+      required: true
+
+  - type: markdown
+    attributes: { value: "---" }
+
+  - type: input
+    attributes:
+      label: Web browser and its version
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Operating system and its version
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: PDF.js version
+    validations:
+      required: true
```

Comment:
Since users often report bugs for very old versions, can we please also add a new *required* Yes/No question along the following lines (and perhaps with a link to https://github.com/mozilla/pdf.js#online-demo)?
`Is the bug present in the latest PDF.js version?`